src/js/components/DataTable/index.d.ts

```diff
@@ -58,9 +60,11 @@ export interface DataTableProps<TRowType = any> {
     | Sections<BackgroundType | string[], BackgroundType, BackgroundType>;
   border?: BorderType | Sections<BorderType>;
   columns?: ColumnConfig<TRowType>[];
+  fill?: boolean | 'vertical' | 'horizontal';
   gridArea?: GridAreaType;
   margin?: MarginType;
   pad?: PadType | Sections<PadType>;
+  pin?: boolean | 'header' | 'footer';
```

Comment:
Duplication of line 43?

Reply:
Nope. Line 43 is for pinning a column for horizontal scrolling. Line 67 is pinning the header and/or footer for vertical scrolling.